src/main/kotlin/lotto/controller/Application.kt

```diff
@@ -0,0 +1,27 @@
+package lotto.controller
+
+import lotto.domain.LotteryChecker
+import lotto.domain.LotteryNumber
+import lotto.domain.Store
+import lotto.domain.EarningsRateCalculator
+import lotto.domain.EarningsRateCalculator.isLoss
+import lotto.view.InputView.getPurchase
+import lotto.view.InputView.getWinningNumbers
+import lotto.view.ResultView.printLotteries
+import lotto.view.ResultView.printStatistics
+import lotto.view.ResultView.printEarningRate
+
+fun main() {
+
+    val purchase = getPurchase()
+    val lotteries = Store.buy(purchase)
+    printLotteries(lotteries)
+
+    val winningNumbers = getWinningNumbers().map { LotteryNumber(it) }
```

Comment:
```suggestion
    val winningNumbers = getWinningNumbers().map(::LotteryNumber)
```
이러한 형태의 lambda는 함수 reference를 넘기는 식으로 구현해볼 수 있습니다. lambda를 사용하는 것과 함수 reference를 넘기는 것의 내부 동작에는 어떤 차이가 있을지 다음 아티클을 참고해보시면 이해하기 편하실거예요! 🙂 
https://proandroiddev.com/kotlin-lambda-vs-method-reference-fdbd175f6845